zulip/integrations/google/doc.md

```diff
@@ -1,72 +1,172 @@
-Get Google Calendar reminders in Zulip! This is a great way to see
-your reminders directly in your Zulip feed.
+# Zulip Google Calendar integration
+
+Get Zulip notifications for your Google Calendar events!
+
+### Create Zulip bot for Google Calendar notifications
+
+{start_tabs}
+
+1.  {!create-a-generic-bot.md!}
 
 1.  {!download-python-bindings.md!}
 
-    This bot should be set up on a trusted machine, because your API
-    key is visible to local users through the command line or config
-    file.
+1.  [Download your bot's `zuliprc` file][download-zuliprc], and save it as
+    `.zuliprc` in your `~/` directory.
+
+[download-zuliprc]: /api/configuring-python-bindings#download-a-zuliprc-file
+
+1.  Optionally, to configure the bot, add a **google-calendar** section to
+    the `.zuliprc` file, like below:
+
+    ```
+    [google-calendar]
+    calendar=primary
+    interval=3600
+    tokens_path=/home/username/zulip/integrations/google-oauth-tokens.json
+    client_secret_path=/home/username/zulip/integrations/client-secret-credentials.json
+    channel=core-team
+    topic=scheduled events
+    noauth_local_webserver=
+    format_message=The event [{title}]({calendar_link}), at {location}, is from {start} to {end}.\n> {description}\n\n[Join call]({google_meet_link}).
+    ```
+
+    See [configuration options](#configuration-options) for the list of
+    available options.
+
+{end_tabs}
+
+### Setup Google OAuth Client ID
+
+{start_tabs}
+
+!!! tip ""
+
+    A free Google account is sufficient for this integration, a Google
+    service account is not required.
+
+1.  In the Google Cloud console, go to
+    **Menu > Google Auth platform > [Clients][clients-menu]**. Click
+    **Create Client**.
+
+[clients-menu]: https://console.cloud.google.com/auth/clients
+
+1.  Select **Application type > Desktop app**. Set **Name** to a name of
+    your choice, such as `Zulip`, and click **Create**. Save the downloaded
+    JSON file as `client_secret.json` in your `~/` directory.
+
+1.  From your Google Cloud dashboard, go to **APIs & Services** and select
+    **OAuth consent screen**. Under **Test users**, click **+ Add Users**.
+    Enter the email address corresponding to your Google Calendar, and click
+    **Save**.
+
+{end_tabs}
+
+### Run the integration script
+
+{start_tabs}
+
+1.  Install the requirements for the integration script with:
+
+    `python {{ integration_path }} --provision`
+
+1.  Run the integration script with:
+
+    `python {{ integration_path }}`
+
+    Authorize access in the browser window that opens, to allow the Zulip
+    bot to view your Calendar. If you've turned on the
```

Comment:
I don't think "turned on" is clear here. Maybe if you've set `noauth_local_webserver` to True? Or something like that.